src/main/config/messages_template.xml

```diff
@@ -450,6 +450,11 @@ See the accompanying LICENSE file for applicable license.
     <response>Ignoring chunk operation.</response>
   </message>
 
+  <message id="DOTJ088E" type="ERROR">
+    <reason>%1</reason>
+    <response></response>
```

Comment:
If we know this is an XML parsing error, should we say so?

```suggestion
    <reason>XML parsing error: </reason>
    <response>%1</response>
```

Reply:
The `%1` in this case is the error message thrown by the XML parser. It describes what the error condition is, like "Attribute value "%50" of type ID must be an NCName when namespaces are enabled.".

This means it's not a response that describes what the user should do next. Because this is a generic parsing, we don't know what the next step should be. It could be "fix your specialization" or "fix your content".

Reply:
OK, so we leave the placeholder in the `<reason>` element, but shall we add the intro for context:

```suggestion
    <reason>XML parsing error: %1</reason>
    <response></response>
```

Reply:
I would prefer to not use a prefix in the reason, but it may be useful for users 🤷🏽. I'll need to get more examples of those messages to help in deciding whether it's helpful of not.

Reply:
Xerces produces error messages listed at https://github.com/apache/xerces2-j/blob/trunk/src/org/apache/xerces/impl/msg/XMLMessages.properties#L230.

Reply:
Added `XML parsing error: ` prefix to the message.